keanu-project/src/main/java/io/improbable/keanu/util/io/MIRSaver.java

```diff
@@ -0,0 +1,70 @@
+package io.improbable.keanu.util.io;
+
+import io.improbable.keanu.network.BayesianNetwork;
+import io.improbable.mir.MIR;
+import io.improbable.mir.SavedBayesNet;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.HashMap;
+import java.util.Map;
+
+public class MIRSaver extends ProtobufSaver {
+
+    public final static String MODEL_NAME = "Saved Keanu Graph";
```

Comment:
Do we want to give the user the option to provide their own model name?

Reply:
added (+ setting of Metadata)